packages/charts/react-charts/stories/src/DeclarativeChart/DeclarativeChartDefault.stories.tsx

```diff
@@ -32,31 +41,103 @@ class ErrorBoundary extends React.Component<ErrorBoundaryProps, ErrorBoundarySta
     ),
   },
+  {
+    key: 'piechart',
+    schema: JSON.parse(
+      '{"visualizer":"plotly","data":[{"type":"pie","marker":{"line":{"color":"#000000","width":2},"colors":["#FEBFB3","#E1396C","#96D38C","#D0F9B1"]},"textfont":{"size":20},"textinfo":"value","hoverinfo":"label+percent","labels":["Oxygen","Hydrogen","Carbon_Dioxide","Nitrogen"],"values":[4500,2500,1053,500]}],"layout":{},"frames":[]}',
+    ),
+  },
+  {
+    key: 'sankeychart',
+    schema: JSON.parse(
+      '{"visualizer":"plotly","data":[{"link":{"color":["rgba(253, 227, 212, 0.5)","rgba(242, 116, 32, 1)","rgba(253, 227, 212, 0.5)","rgba(219, 233, 246, 0.5)","rgba(73, 148, 206, 1)","rgba(219, 233, 246,0.5)","rgba(250, 188, 19, 1)","rgba(250, 188, 19, 0.5)","rgba(250, 188, 19, 0.5)","rgba(127, 194, 65, 1)","rgba(127, 194, 65, 0.5)","rgba(127, 194, 65, 0.5)","rgba(211, 211, 211, 0.5)","rgba(211, 211, 211, 0.5)","rgba(211, 211, 211, 0.5)"],"value":[20,3,5,14,1,1,3,17,2,3,9,2,5,9,8],"source":[0,0,0,1,1,1,2,2,2,3,3,3,4,4,4],"target":[5,6,7,5,6,7,5,6,7,5,6,7,5,6,7]},"node":{"pad":10,"line":{"color":"black","width":0},"color":["#F27420","#4994CE","#FABC13","#7FC241","#D3D3D3","#8A5988","#449E9E","#D3D3D3",null,null,null,null,null,null,null],"label":["Remain+No – 28","Leave+No – 16","Remain+Yes – 21","Leave+Yes – 14","Didn’t vote in at least one referendum – 21","46 – No","39 – Yes","14 – Don’t know / would not vote"],"thickness":30},"type":"sankey","domain":{"x":[0,1],"y":[0,1]},"orientation":"h","valueformat":".0f"}],"layout":{"font":{"size":10},"title":"Scottish Referendum Voters who now want Independence","height":772},"frames":[]}',
```

Comment:
Instead of defining the json data here can we download the data from the url dynamically